### ISSUE TITLE
Import primary model functions for forest growth

### ISSUE THREAD

**Issue by jpy-luke** [menu-hanke/sim-workbench#3]
Primary domain model functions need to be ported from possibly https://github.com/menu-hanke/LAME or another source. Discuss separately with @artohaa and @Sorvalampi which functions should these be. These functions should work with data structures parsed in #2.

UPDATE:
* control.yaml now has a structure operation_params for passing parameters to model operations
* the `forestry` package file `operations.py` is the entry point for models
* operation functions must take a ForestStand as first parameter, and a dict for model specific parameters (from operation_params) as the second parameter
* `forestry_operations_test.py` contains a stub example of how to start with unit tests for model operations

In other words: implementation for models is ready to begin

**Comment by jpy-luke** [comment]
Closing as overly broad as an issue. Create specific tasks for specific models.
